sale_commission/models/settlement.py

```diff
@@ -172,8 +179,11 @@ class SettlementLine(models.Model):
         comodel_name="res.partner", readonly=True, related="agent_line.agent",
         store=True)
     settled_amount = fields.Float(
-        related="agent_line.amount", readonly=True, store=True)
+        string='Settled Amount', digits_compute=dp.get_precision('Account'))
```

Comment:
Hi @pedrobaeza, thank you for review, the reason is the related field has the total value to be paid to commission but now with partial payments this field can't be the same because here the value can be 30%, 50% etc depending the prorportional of what was paid in invoice